src/server/services/webhookEventService/notificationHandlers/projectScreenVersionHandler/projectScreenVersionHandler.ts

```diff
@@ -17,16 +17,21 @@ class ProjectScreenVersionHandler extends NotificationHandler<ProjectScreenVersi
         const [{
             context: {
                 screen: {
-                    name: screenName
+                    name: screenName,
+                    variant: screenVariant
                 },
                 project: {
                     name: projectName
                 }
             }
         }] = events;
-        return events.length === 1
-            ? md`**${screenName}** is updated in _${projectName}_! ${getRandomEmoji()}`
-            : md`**${events.length} screens** are updated in _${projectName}_! ${getRandomEmoji()}`;
+
+        if (events.length > 1) {
+            return md`**${events.length} screens** are updated in _${projectName}_! ${getRandomEmoji()}`;
```

Comment:
We can also add the `screen.variant.group.name` to the copy when the variant group id is same for all events.